application/src/main/resources/application.yml

```
@@ -0,0 +1,8 @@
+spring:
+  datasource:
+    url: jdbc:postgresql://localhost:5432/stock_analysis?currentSchema=stock_analysis
+    username: stockanalysisservice
+    password: ywY3a2to
+  liquibase:
+    change-log: file:application/etc/database/changelog.yml
+    enabled: true
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)